packages/ui/src/Button/index.tsx

```diff
@@ -133,7 +136,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
 
           'relative',
           'text-neutral-contrast',
-          'flex items-center justify-center',
+          'inline-flex items-center justify-center',
```

Comment:
issue: this is a breaking change we shouldn't introduce for such purpose. 

suggestion: from button's parent, apply `[&>button]:inline-flex` or `[&_button]:inline-flex` class. Might feel hacky, but it's Tailwind – would look nicer with plain css.

Reply:
Ok will do that